addons/website/static/src/snippets/s_blockquote/001.scss

```diff
@@ -36,8 +36,11 @@
     // Cover
     &.s_blockquote_cover {
         text-align: center;
-        .s_blockquote_icon + .s_blockquote_content {
-            margin-top: -$spacer * 1.5;
+        .s_blockquote_icon {
+            float: none !important;
```

Comment:
To avoid the important rule, can't you remove the float class from the dom and have a float rule for the non-cover style instead?